libraries/engage/page/widgets/ImageSlider/hooks.js

```diff
@@ -84,36 +88,38 @@ export const useImageSliderWidget = () => {
       },
     };
 
+    const showPagination = images.length > 1;
+    // Create a key that changes when relevant config changes, to force remount of Swiper
+    const componentKey = JSON.stringify({
```

Comment:
this is only needed in preview mode as the original comment stated. inside the PWA settings will never change during runtime.